app/partials/app.cjsx

```diff
@@ -47,6 +50,7 @@ PanoptesApp = createReactClass
 
   componentWillMount: ->
     @geordiLogger = new GeordiLogger
+    @geordiLogger.subscribe(new GeordiLogAdapter(), new GALogAdapter(window.dataLayer)
```

Comment:
This needs a closing parentheses

Reply:
oh for goodness-- hang on one second lol